react/features/toolbar/actions.native.js

```diff
@@ -0,0 +1,218 @@
+/* @flow */
+
+import type { Dispatch } from 'redux-thunk';
+
+import {
+    SET_ALWAYS_VISIBLE_TOOLBAR,
+    SET_SUBJECT_SLIDE_IN,
+    SET_TOOLBAR_BUTTON,
+    SET_TOOLBAR_IS_HOVERED,
+    SET_TOOLBAR_VISIBILITY
+} from './actionTypes';
+
+/**
+ * Event handler for local raise hand changed event.
+ *
+ * @param {boolean} isRaisedHand - Flag showing whether hand is raised.
+ * @returns {Function}
+ */
+export function changeLocalRaiseHand(isRaisedHand: boolean): Function {
+    return (dispatch: Dispatch<*>, getState: Function) => {
+        const buttonName = 'raisehand';
+        const state = getState();
+        const { secondaryToolbarButtons } = state['features/toolbar'];
+        const button = secondaryToolbarButtons.get(buttonName);
+
+        button.toggled = isRaisedHand;
+
+        dispatch(setToolbarButton(buttonName, button));
```

Comment:
I don't like this naming because it raises unnecessary complexity and that's why there's a rule in our coding style that 1 abstraction should be given 1 name. I cannot understand why you call it `buttonName` here and `key` in `setToolbarButton`. So when I go and try to understand how the whole thing works I have to keep a map of aliases for one and the same abstraction.